### ISSUE TITLE
nextCursor doesn't exist's.

### ISSUE THREAD

**Issue by joaoarmando** [robertodoering/twitter_api#2]
Hello Dear, great lib! Thanks for it.
I have an issue, we must pass the parameter "nextCursor" (int) to paginate on friendsList() but this method doesn't return the nextCursor parameter, only return nextCursorStr witch in my case returns Null too. I'm trying to do it by my self but i'm not having success. Could you please take a look?

**Comment by joaoarmando** [comment]
Fixed by adding in paginated_users.dart
```
  @JsonKey(name: 'next_cursor')
  int nextCursor;
  @JsonKey(name: 'next_cursor_str')
  String nextCursorStr;
  @JsonKey(name: 'previous_cursor')
  int previousCursor;  
  @JsonKey(name: 'previous_cursor_str')
  String previousCursorStr;
```

I've tried to create a new branch and submit a merge request but i couldn't. I'm always getting 403 forbidden from Github.

**Comment by robertodoering** [comment]
Hi @joaoarmando, thank you very much for reporting this issue!
The bug here was the missing parameter `fieldRename: FieldRename.snake` in the `@JsonSerializable` annotation for the model.
This automatically converts the field names, such as `nextCursorStr` into `next_cursor_str` for the response without the need of the `@JsonKey` annoations.

I only add the string representations of the cursors as the int can exceed the max integer size for javascript and therefore cause problems when using this package for the web.

I have released a `0.2.1` with the fix, `nextCursorStr` should no longer be `null` in your response now.
Thanks again and please let me know if you stumble across any more issues :)
